### ISSUE TITLE
Ignite.Blazor Add custom Actions to Row Actions in Grid

### ISSUE THREAD

**Issue by GuyDavidBanks** [IgniteUI/igniteui-blazor#96]
I was informed by support that is is not possible to add custom actions to the Grid Row Ations popup toolbar in ignite.Blazor, but it is supported in Angular.  I would like to add an action to navigate to a child page etc.

Please could this be added to the Row actions as in Angular.

**Comment by mddifilippo89** [comment]
@dkamburov @gmurray81 @ig-robstoffers 
There appears to be some confusion around this feature. It was marked completed here but I don't think this feature was ever completed/merged in Blazor

https://github.com/IgniteUI/igniteui-xplat-docs/issues/746

**Comment by dkamburov** [comment]
@mddifilippo89 Yeah, there might be a confusion arround it. For example editing options for adding and removing rows are working as expected.
```
  <IgbActionStrip
  Name="actionStrip"
  @ref="actionStrip">
      <IgbGridPinningActions>
      </IgbGridPinningActions>

      <IgbGridEditingActions
      EditRow="true"
      DeleteRow="true"
      AddRow="true">
      </IgbGridEditingActions>

  </IgbActionStrip>
```

But there seems to be an issue with custom elements - like adding a button to the content of the action strip for example. This behavior can be observed in WebComponents as well

**Comment by mddifilippo89** [comment]
@dkamburov Can you reassign this to me when this merged to main so I can test it and add the doc about it?
